### PR TITLE
Added "--status" to unprivileged options

### PR DESCRIPTION
**Pull request by roguepullrequest**
Felt that `--status` `-status` `status` shouldn't need root privileges to run. 
There is nothing in the commands that root access is needed. Privacy isn't a concern here as this is info available on the system.

### CODE REVIEW

File: protonvpn-cli.sh [hunk 1]
```diff
@@ -11,6 +11,7 @@ version=1.1.2
 
 if [[ ("$UID" != 0) && ("$1" != "ip") && ("$1" != "-ip") && \
       ("$1" != "--ip") && ! (-z "$1") && ("$1" != "-h") && \
+      ("$1" != "--status") && ("$1" != -status") && ("$1" != "status") &&\
```
Comment (vitorgalvao) [inline protonvpn-cli.sh]:
```suggestion
      ("$1" != "--status") && ("$1" != -status") && ("$1" != "status") && \
```

Comment (roguepullrequest) [reply protonvpn-cli.sh]: Thanks... with this saw missing quotes. Commited changes.

**Comment by kaplun** [comment]
Hi @roguepullrequest,

I am taking over from @mazen160 for integrating code. This PR looks good to me. Thanks for the contribution!

**Comment by Tamaranch** [comment]
This PR introduces a regression: we need root privileges to read `"$(get_protonvpn_cli_home)/.connection_config_id"` at [line 701](https://github.com/ProtonVPN/protonvpn-cli/blob/a4bdd7861a0b0d880c6d2f52dd2e6b0837e9c24c/protonvpn-cli.sh#L701).

**Pull request merged**